scipy/_distributor_init.py

```diff
@@ -0,0 +1,19 @@
+""" Distributor init file
+
+Distributors: you can add custom code here to support particular distributions
+of scipy.
+
+For example, this is a good place to put any checks for hardware requirements.
+
+The scipy standard source distribution will not put code in this file, so you
+can safely replace this file with your own version.
```

Comment:
Ah yes - right - the empty file (with documentation) should go in the scipy repo, and the Windows build should add the relevant code.  Or the code should go somewhere else.